staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -331,6 +331,8 @@ func (s *GenericAPIServer) InstallAPIGroup(apiGroupInfo *APIGroupInfo) error {
 	return nil
 }
 
+// AddAPIGroupForDiscovery adds the specified group to the list served to discovery queries.
+// Discovery lists groups in order of registration.
```

Comment:
Is registration the right term here? I don't see it used else where, and perhaps something like "... in the order in which they were added" may be clearer.